packages/office-ui-fabric-react/src/components/Button/Button.test.tsx

```diff
@@ -4,277 +4,245 @@ import * as ReactDOM from 'react-dom';
 /* tslint:enable:no-unused-variable */
```

Comment:
Not sure why the dif shows so many changes, but I only added lines 20 - 74 and wrapped all tests inside `describe('Button')`